### PR TITLE
kmehr fixes

### PR DESCRIPTION
**Pull request by dorian-figac**

### CODE REVIEW

File: src/main/java/org/taktik/icure/entities/base/Code.java [hunk 1]
```diff
@@ -64,6 +64,8 @@
 
 	protected String data;
 
+    protected String parent; //ID of the parent
+
```
Comment (aduchate) [inline src/main/java/org/taktik/icure/entities/base/Code.java]: Malheureusement, parent n'est plus supporté... Il faut utiliser les qualifiedLinks

File: src/main/java/org/taktik/icure/dao/impl/PatientDAOImpl.java [hunk 2]
```diff
@@ -59,11 +56,11 @@ public PatientDAOImpl(@SuppressWarnings("SpringJavaAutowiringInspection") @Quali
         initStandardDesignDocument();
     }
 
-    @Override
-    @View(name = "by_hcparty_name", map = "classpath:js/patient/By_hcparty_name_map.js", reduce = "_count")
```
Comment (aduchate) [inline src/main/java/org/taktik/icure/dao/impl/PatientDAOImpl.java]: This is a breaking change... Why would you want to do that

Comment (dorian-figac) [reply src/main/java/org/taktik/icure/dao/impl/PatientDAOImpl.java]: this is from a commit by Bogdan Safta https://github.com/AssossCare/icure-backend/commit/8a13b0a36deffdec344017cbc0440cf4aff41213 I don't know more

File: src/main/java/org/taktik/icure/entities/base/Code.java (hunk 1 above)
Comment (dorian-figac) [reply src/main/java/org/taktik/icure/entities/base/Code.java]: removed in last commit

**Pull request closed without merging**